app/dashboard/templates/dashboard/domain_detail/dns.html

```diff
@@ -28,15 +28,15 @@ <h1 class="h3"> {{ custom_domain.domain }} </h1>
       </div>
 
       <div class="mb-2">Add the following MX DNS record to your domain. <br>
-        Please note that there's a point (<em>.</em>) at the end target addresses. <br>
-        Also some domain registrars (Namecheap, CloudFlare, etc) might use <em>@</em> for the root domain.
+        Please note that there's a point (<em class="text-default">.</em>) at the end target addresses. <br>
```

Comment:
do you think there's a way to apply `.text-default` automatically to `<em>`?

Reply:
Yep, we can apply the style to the em directive globally. I am going to update it